src/Modules/ModulesGraded.jl

```diff
@@ -1926,3 +1926,96 @@ function hom(F::FreeMod_dec, G::FreeMod_dec)
   set_attribute!(GH, :show => Hecke.show_hom, :hom => (F, G), :module_to_hom_map => to_hom_map)
   return GH, to_hom_map
 end
+
+#############truncation#############
+
+@doc raw"""
+    truncate(M::ModuleFP, g::GrpAbFinGenElem, task::Symbol = :with_morphism)
+
+Given a finitely presented graded module `M` over a $\mathbb Z$-graded multivariate 
+polynomial ring with positive weights, return the truncation of `M` at degree `g`.
+
+Put more precisely, return the truncation as an object of type `SubquoModule`. 
+
+Additionally, if `N` denotes this object,
+
+- return the inclusion map `N` $\to$ `M` if `task = :with_morphism` (default),
+- return and cache the inclusion map `N` $\to$ `M` if `task = :cache_morphism`,
+- do none of the above if `task = :none`.
+
+If `task = :only_morphism`, return only the inclusion map.
+
+    truncate(M::ModuleFP, d::Int, task::Symbol = :with_morphism)
+
+Given a module `M` as above, and given an integer `d`, convert `d` into an element `g`
+of the grading group of `base_ring(I)` and proceed as above.
+
+# Examples
+```jldoctest
+julia> R, (x, y, z) = graded_polynomial_ring(QQ, ["x", "y", "z"]);
+
+julia> F = graded_free_module(R, 1)
+Graded free module R^1([0]) of rank 1 over R
+
+julia> V = [x*F[1]; y^4*F[1]; z^5*F[1]];
+julia> M, _ = quo(F, V);
+
+julia> M[1]
+e[1]
+
+julia> MT = truncate(M, 3);
+
+julia> MT[1]
+Graded subquotient of submodule of F generated by
+1 -> z^3*e[1]
+2 -> y*z^2*e[1]
+3 -> y^2*z*e[1]
+4 -> y^3*e[1]
+5 -> x*z^2*e[1]
+6 -> x*y*z*e[1]
+7 -> x*y^2*e[1]
+8 -> x^2*z*e[1]
+9 -> x^2*y*e[1]
+10 -> x^3*e[1]
+by submodule of F generated by
+1 -> x*e[1]
+2 -> y^4*e[1]
+3 -> z^5*e[1]
```

Comment:
The output makes sense, but it is for F/V, where you basically do nothing else , but writing down the monomials of the desired lowest degree in F as generators. Maybe you want to illustrate truncate also with sub(F,V)[1], to parallel the explanations in the ideal case, where we really see something happening.